idalib/src/idb.rs

```diff
@@ -213,6 +214,36 @@ impl IDB {
         }
     }
 
+    pub fn get_cmt(&self, ea: Address, rptble: bool) -> String {
```

Comment:
Similar to `append_cmt`/`set_cmt`, I'd create a second variation of this API with a default value for non-repeatable comments.